reasoning_gym/games/rush_hour.py

```diff
@@ -182,7 +184,7 @@ def score_answer(self, answer: Optional[str], entry: dict) -> float:
             board.perform_moves(answer)
 
             # Check if solved
-            return 1.0 if board.solved else 0.0
+            return 1.0 if board.solved else 0.01
```

Comment:
I had removed the minimal reward from most of the envs. I think we should keep the score function as simple as possible.